readme.md

```diff
@@ -1 +1 @@
-﻿
\ No newline at end of file
+﻿https://Alex-Edward-Klim.github.io/codejam-canvas
```

Comment:
Please add link to demo to  MR description next time

Reply:
Totally agree with you.
But I must say that I just strictly followed these given rules:
https://github.com/rolling-scopes-school/docs/blob/master/docs/stage2.md
And as there isn't any demo link in these ruleset, I decided not to take a risk, so that's why I didn't add a demo link. BUT I wanted to! :) Next time I definately will do.